numpy/core/src/npysort/quicksort.c.src

```diff
@@ -107,6 +111,7 @@ quicksort_@suff@(void *start, npy_intp num, void *NOT_USED)
                 *sptr++ = pi - 1;
                 pl = pi + 1;
             }
+            *psdepth++ = cdepth + 1;
```

Comment:
doh right I moved this variable into a higher scope ...
it does not really matter as we don't really care about the absolute depth of the stack we just want to cut of at some point
But it is relevant to avoid the overflow of the bounded to 100 elements stack which could happen for arrays greater than 2^50 elements so I'll change it
sorry for being so dense here